### PR TITLE
Mttdoc 237 network time

### PR DESCRIPTION
**Pull request by Briancoughlin**
**Select the type of change:**

- [ ] Small Changes - Typos, formatting, slight revisions
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site and Docusaurus

**Release:** 
🚀  Develop - in progress development

**Purpose of the Pull Request:**
This adds documentation to support the  RFC's for network time (https://github.com/Unity-Technologies/com.unity.multiplayer.rfcs/pull/14)  and Synced Network Variables (https://github.com/Unity-Technologies/com.unity.multiplayer.rfcs/pull/12)

**Issue Number:** 

[MTTDOC-39 SDK Tick Based System](https://jira.unity3d.com/browse/MTTDOC-39)
[MTTDOC-237 NetworkTime](https://jira.unity3d.com/browse/MTTDOC-237)
[MTTDOC-238 Synced Network Variables](https://jira.unity3d.com/browse/MTTDOC-238)

### CODE REVIEW

File: docs/components/time.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+var time = NetworkManager.Singleton.NetworkTime;
+```
+use
+```csharp
+var time = NetworkManager.Singleton.PredictedTime.Time;
+```
+
+## New Features
+
+-->
```
Comment (lkrell) [inline docs/components/time.md:35]: Why is this hidden? Being saved for something specific?

**Review by lkrell — COMMENTED** [review]
Quick feedback

**Comment by lkrell** [comment]
I'll review when this is ready to review. Thanks!

File: docs/components/time.md (hunk 1 above)
Comment (Briancoughlin) [reply docs/components/time.md:35]: its being hidden as singleton is being removed

**Review by chrispope — APPROVED** [review]
Awesome!

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.multiplayer.docs?pullRequest=189) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.multiplayer.docs?pullRequest=189) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.cds.internal.unity3d.com/check/Unity-Technologies/com.unity.multiplayer.docs?pullRequest=189) it.</sub>

**Comment by Briancoughlin** [comment]
This work has now been overtaken by MTTDOC-321

**Pull request closed without merging**